src/makeInfinityAdder.js

```diff
@@ -5,6 +5,23 @@
  */
 function makeInfinityAdder() {
   // write code here
+  let sum = 0;
+
+  const adder = (b) => {
+    if (b === undefined) {
+      const finalSum = sum;
+
+      sum = 0;
+
+      return finalSum;
```

Comment:
The check for 'undefined' is correct for the functionality, but it would be more robust to also ensure that the function is called without any arguments. This can be done by checking the 'arguments.length' property.